examples/mass_delete_old_medias.py

```diff
@@ -0,0 +1,562 @@
+        )
+    else:
+        for oid, media in medias.items():
+            logger.debug(f'[Dry run] Media {ms_url}{oid} would have been deleted.')
+            deleted_count += 1
+            deleted_size += media['storage_used']
+        logger.info(
+            f'[Dry run] {deleted_count} medias ({pp_size(deleted_size)}) would have been have been deleted.'
+        )
+
+
+class ArgumentParser(argparse.ArgumentParser):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.add_argument(
+            '--conf',
+            help='Path to the configuration file (e.g. myconfig.json).',
+            required=True,
+            type=str,
+        )
+        self.add_argument(
+            '--delete-date',
+            help='Date after which content will be deleted when running '
+                 'the script, e.g. "2024-10-28"; running this script '
+                 'before this date will result in notifying users, '
+                 'otherwise it will delete the medias.',
+            type=str,
+            required=True,
+        )
+        self.add_argument(
+            '--added-before',
+            help='Maximum "add_date" for a media to be considered for '
+                 'deletion (e.g. "2022-01-28"). Any media added on or '
+                 'after this date will not be considered for deletion.',
+            type=str,
+            required=True,
+        )
+        self.add_argument(
+            '--skip-category',
+            help='Category name used to signify that content must be preserved.',
+            type=str,
+            default='do not delete',
+        )
+        self.add_argument(
+            '--html-email-template',
+            help='Path to HTML email template file. The template will be populated at '
+                 'runtime with dynamic values via these 4 variables: media_count, '
+                 'delete_date, skip_category, list_of_media. Your template should use '
+                 'Python new-style formatting syntax '
+                 '(e.g.: "You have until {delete_date} to review each media").',
+            type=Path,
+            default='./email.html',
+        )
+        self.add_argument(
+            '--plain-email-template',
+            help='Path to plain email template file. This plain variant will be displayed '
+                 'to users whose email client is set to prevent HTML in emails. The template '
+                 'variables available are the same as for the "--html-email-template" argument.',
+            type=Path,
+            default='./email.txt',
+        )
+        self.add_argument(
+            '--email-subject-template',
+            help='Template string to use for the email subject line. The template '
+                 'variables available are the same as for the "--html-email-template" argument.',
+            type=str,
+            default='Action required: {media_count} medias will be deleted on {delete_date}',
```

Comment:
The email subject must mention the fact that we are talking about the video platform, and which one

```suggestion
            default='Action required on the video platform {platform_url}: {media_count} medias will be deleted on {delete_date}',
```